compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneRegionalDisksFromConsistencyGroup.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute.disks.consistencygroup;
+
+// [START compute_consistency_group_clone_regional_disk]
+import com.google.cloud.compute.v1.BulkInsertDiskResource;
+import com.google.cloud.compute.v1.BulkInsertRegionDiskRequest;
+import com.google.cloud.compute.v1.Operation;
+import com.google.cloud.compute.v1.Operation.Status;
+import com.google.cloud.compute.v1.RegionDisksClient;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CloneRegionalDisksFromConsistencyGroup {
+
+  public static void main(String[] args)
+          throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Project ID or project number of the Cloud project you want to use.
+    String project = "YOUR_PROJECT_ID";
+    // Region in which your disks and consistency group are located.
+    String region = "us-central1";
+    // Name of the consistency group you want to clone disks from.
+    String consistencyGroupName = "YOUR_CONSISTENCY_GROUP_NAME";
+
+    cloneRegionalDisksFromConsistencyGroup(project, region, consistencyGroupName);
+  }
+
+  // Clones regional disks from a consistency group.
+  public static Status cloneRegionalDisksFromConsistencyGroup(
+          String project, String region, String consistencyGroupName)
+          throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    String sourceConsistencyGroupPolicy = String.format(
+            "projects/%s/regions/%s/resourcePolicies/%s", project, region, consistencyGroupName);
```

Comment:
The project, region here are different from the ones in line 56 and 57. 

This refers to the consistency group project and region, whereas line 56 and 57 refers to where the disks should be cloned in.

Kindly modify the samples to get these extra inputs and make changes accordingly.

Reply:
There is no mistake. The disk region and consistency group must be the same. Here is a merged sample in Python-  https://github.com/GoogleCloudPlatform/python-docs-samples/blob/9531bf469545820624a2ef0fa1a2aaa965dc2c99/compute/client_library/snippets/disks/%D1%81onsistency_groups/clone_disks_consistency_group.py#L22